src/App.tsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import 'styled-components/macro';
+import ReactTooltip from 'react-tooltip';
+import styled from 'styled-components/macro';
 import type {} from 'styled-components/cssprop';
```

Comment:
Do we need this here?

Reply:
Yeah, it's odd, but you have to import it once in your project apparently to avoid errors in Typescript: https://styled-components.com/docs/api#usage-with-typescript